### DISCUSSION TITLE
Add the possibility to create events that use an external ticket registration

### DISCUSSION THREAD

**Discussion by ZPascal** [pretix/pretix#3771] [Ideas]
We're currently missing the feature, to add new events that use an external ticket registration. A requirement from our side is in general the possibility to add new events that use an external ticket registration.

@raphaelm What do think about this option?

**Comment by raphaelm** [comment]
You can always create a product with no products and add a link to the text. I don't think having this as a first-class feature makes sense, because pretix is, after all, a ticket sales system ;)
